cmd/thermal/main.go

```diff
@@ -12,13 +12,16 @@ import (
 	"io/ioutil"
 	"log"
 	"os"
+	"time"
 
 	"periph.io/x/periph/conn/physic"
 	"periph.io/x/periph/host"
 	"periph.io/x/periph/host/sysfs"
 )
 
 func mainImpl() error {
+	cont := flag.String("cont", "", "Reads continuously from the sensor with this name")
```

Comment:
I think it should be `-dev` and looks as:
- If not specified, read all devices found
- If specified, read only this device

If you want to simplify your life, you could require `-dev` when using `-interval`.

Reply:
Done, sorry for the delay again!